index.html

```diff
@@ -781,6 +781,35 @@ <h3>Examples</h3>
     </div>
     <!-- ends Crop video -->
 
+    <!-- Change video colour to black and white -->
+    <label class="recipe" for="col_change">Change video colour to black and white</label>
+    <input type="checkbox" id="col_change">
+    <div class="hiding">
+      <h3>Change video colour to black and white</h3>
+      <p><code>ffmpeg -i <em>input_file</em> -filter_complex hue=s=0 -c:a copy <em>output_file</em></code></p>
+      <p>A basic command to alter colour hue to black and white using filter_complex (credit @FFMPEG via Twitter).</p>
+      <dl>
+        <dt>ffmpeg</dt><dd>starts the command</dd>
+        <dt>-i <em>input_file</em></dt><dd>path, name and extension of the input file</dd>
+        <dt>-filter_complex hue=s=0</dt><dd>uses filter_complex command to set the hue to black and white</dd>
+        <dt>-c:a copy</dt><dd>copies the encode settings of the input_file to the output_file</dd>
+        <dt><em>output_file</em></dt><dd>path, name and extension of the output_file</dd>
+      </dl>
+      <p>An alternative that preserves interlacing information for a prores 422hq file generated, for example, from a tape master (credit Dave Rice):</p>
+      <p><code>ffmpeg -i <em>input_file</em> -c:v prores_ks -flags +ildct+ilme -map 0 -c:a copy -profile:v 3 -vf hue=s=0 <em>output_file</em></code></p>
+      <dl>
+        <dt>ffmpeg</dt><dd>starts the command</dd>
+        <dt>-i <em>input_file</em></dt><dd>path, name and extension of the input file</dd>
+        <dt>-c:v prores_ks</dt><dd>encodes the video to ProRes (prores_ks marks the stream as interlaced, unlike prores)</dd>
+        <dt>-flags +ildct+ilme</dt><dd>ensures that the output_file has interlaced field encoding, using interlace aware discrete cosine transform and interlace aware motion estimation</em></dd>
```

Comment:
<3 +1 for explaning what ildct and ilme mean

Reply:
+ilme is not used by ProRes as it is intra only.

Reply:
No problem, @b00giehead, already fixed.